addons/mail_group/tests/test_mail_group_message.py

```diff
@@ -122,3 +122,35 @@ def test_email_empty_from(self):
 
         mails = self.env['mail.mail'].search([('subject', '=', 'Test subject')])
         self.assertEqual(len(mails), 0, "Email should not be delivered when no email is specified")
+
+    def test_batch_emails(self):
+        """Test that when someone sends an email to a large group that it is
+        delivered exactly to those people"""
+        self.test_group.write({
+            'access_mode': 'members',
+            'alias_contact': 'followers',
+            'moderation': False,
+        })
+        self.test_group.member_ids.unlink()
+
+        for num in range(1932):  # year of Emu war
```

Comment:
or to avoid creating 1000+ members and messages, we lower `GROUP_SEND_BATCH_SIZE` during the test, so we can get around with having just a dozen of members?

unless you wanted to also test the performance aspect?

PS: it may not be entirely crazy to make the batch size configurable via an ICP, if we want a cleaner override option. Like `mail.session.batch.size` (which we could even reuse here?)

Reply:
+1 for the configurable parameter (instead of the global variable)

Reply:
Ok I will change that in master and reduce the test size (sadely losing my historical reference)